lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -33,6 +33,7 @@ click_button:
   element:
     selector:
       xpath: //button[contains(.,'<button_text>')]
+    timeout: 10
```

Comment:
Create button when create an operator sometimes comes across an instant error message and stuck for a second when running auto, It's failed almost 100%. The button works after 1~2s, need to wait for a few seconds here for stable. Step could be passed stable now.